packages/suite/src/views/wallet/account/summary/components/AccountHeader/index.tsx

```diff
@@ -50,20 +35,14 @@ const AccountHeader = ({ account, network, fiatRates, localCurrency, isHidden }:
         account.networkType === 'ripple' && !account.empty && account.misc && account.misc.reserve
             ? account.misc.reserve
             : '0';
+    const accountNameMessage =
+        account && account.networkType === 'ethereum'
+            ? l10nSummaryMessages.TR_NETWORK_AND_TOKENS
+            : undefined;
     return (
         <>
             <AccountHeading>
-                <AccountName>
-                    <StyledCoinLogo size={24} symbol={account.symbol} />
-                    <AccountTitle>
-                        <FormattedMessage
-                            {...(account.imported
-                                ? l10nCommonMessages.TR_IMPORTED_ACCOUNT_HASH
-                                : l10nCommonMessages.TR_ACCOUNT_HASH)}
-                            values={{ number: String(account.index + 1) }}
-                        />
-                    </AccountTitle>
-                </AccountName>
+                <AccountName account={account} message={accountNameMessage} />
```

Comment:
elements are not aligned properly
![Screenshot 2019-10-17 at 15 52 56](https://user-images.githubusercontent.com/3435913/67017092-84614180-f0f9-11e9-8c91-084e9d96ffaa.png)